x-pack/plugins/grokdebugger/server/index.js

```diff
@@ -5,8 +5,10 @@
  * 2.0.
  */
 
-import { Plugin } from './plugin';
+import { Plugin, config } from './plugin';
 
-export function plugin(initializerContext) {
-  return new Plugin(initializerContext);
+export function plugin() {
+  return new Plugin();
 }
+
+export { config };
```

Comment:
Not sure that this needs to be exported anymore

Reply:
This is still needed to export the config schema for the plugin. without exporting this you get the following error:
```
[FATAL][root] Reason: No validation schema has been defined for [xpack.grokdebugger]
Error: No validation schema has been defined for [xpack.grokdebugger]
    at ConfigService.isEnabledAtPath (config_service.ts:188:13)
    at PluginsService.handleDiscoveredPlugins (plugins_service.ts:305:10)
    at PluginsService.discover (plugins_service.ts:115:5)
    at Server.preboot (server.ts:172:30)
    at Root.preboot (index.ts:57:14)
    at bootstrap (bootstrap.ts:98:25)
    at Command.<anonymous> (serve.js:295:5)

 FATAL  Error: No validation schema has been defined for [xpack.grokdebugger]
 ```